config/product/cis.js

```diff
@@ -138,7 +138,7 @@ export function init(store) {
       name:          'isDefault',
       labelKey:      'tableHeaders.default',
       formatter:     'IconText',
-      formatterOpts: { getIcon: row => row.spec.customBenchmarkConfigMapName ? 'icon-x icon-2x' : 'icon-checkmark icon-2x' }
+      formatterOpts: { getIcon: row => row.isCustom ? 'icon-checkmark icon-2x' : 'icon-x icon-2x' }
```

Comment:
This (well the previous commit that added getIcon) is causing a warning in SSR because the definition gets put into the store and then serialized to the client, but functions cant't be turned into JSON.  

>  WARN  Cannot stringify a function getIcon                                                                                                             01:32:00

Consider using the existing `checked` formatter.